developerTools/scriptReferences/entityClientScript/entityClientScript.js

```diff
@@ -0,0 +1,112 @@
+//
+// entityClientScript.js
+//
+// Adapted from script https://github.com/robin-k-wilson/hifi-dev-utils/blob/master/client_entity_all_events.js
+// Updated by Robin Wilson 4/15/2019
+//
+// Copyright 2019 High Fidelity, Inc.
+//
+// Distributed under the Apache License, Version 2.0.
+// See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function() {
+    // example static variable
+    var I_AM_STATIC_VARIABLE = 200; 
+    var that = null;
+
+    function EntityClientScript() {
+        that = this;
+        // dynamic variables for each instance of the script
+        this.entityID;
```

Comment:
please remove this line - you don't need it, and it's not actually performing any operations